specification/glossary.md

```diff
@@ -0,0 +1,44 @@
+# Glossary
+
+Below are a list of some OpenTelemetry specific terms that are used across this
+specification.
+
+## (Telemetry) Library
```

Comment:
I think it will be very easy to confuse "Telemetry Library" with "Instrumenting Library." Is there some way we can make this naming less ambiguous?

Reply:
I know these are ambiguous, hence the glossary ;-)
I'm happy about suggestions though.

Reply:
Yep, that's the name currently used in the project.